src/main/zapHomeFiles/hudtutorial/en_GB/Comments.html

```diff
@@ -24,7 +24,7 @@ <h1>Comments</H1>
 	Hovering over these icons will show the related comment and clicking on them will copy the comment to the clipboard.
 	<p>
 	Showing comments may well mess up the formatting of the target page, and in some cases may
-	break functionality if there are any scripts that rely on the DOM structure not changing. 
+	break functionality if there any scripts rely on the DOM structure not changing. 
```

Comment:
oops, the `there` before `any` was supposed to be dropped. *sigh*

Reply:
Yup, it would make sense that way 👍